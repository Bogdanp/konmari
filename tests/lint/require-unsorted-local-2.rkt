#lang racket/base

(require racket/string
         "b.rkt"
         racket/format
         racket/file
         "a.rkt")
