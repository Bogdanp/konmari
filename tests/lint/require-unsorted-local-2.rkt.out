/require-unsorted-local-2.rkt:4:10:warning:relative require "b.rkt" should come after "racket/format"
/require-unsorted-local-2.rkt:6:10:warning:"racket/file" should come before "racket/format"
